src/natcap/invest/spec.py

```diff
@@ -2375,7 +2365,7 @@ def describe_arg_from_spec(name, spec):
             indented_block += spec.format_rst()
 
     elif type(spec) is CSVInput:
-        if not spec.columns and not spec.rows:
+        if not spec.columns:
             first_line += gettext(
                 ' Please see the sample data table for details on the format.')
```

Comment:
Now this has me wondering if we have a weird HRA table or something that does not have `rows` or `columns`, and maybe my earlier comments about making `columns` a required attribute is not quite possible 🫤